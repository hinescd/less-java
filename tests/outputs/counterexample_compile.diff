--- tests/outputs/counterexample_compile.out	2019-02-04 13:05:51.937399291 -0500
+++ tests/outputs/counterexample_compile.exp	2019-02-04 13:05:52.194400676 -0500
@@ -1,23 +0,0 @@
-:generateGrammarSource UP-TO-DATE
-:compileJava UP-TO-DATE
-:processResources NO-SOURCE
-:classes UP-TO-DATE
-:ljwrappers:compileJava UP-TO-DATE
-:ljwrappers:processResources NO-SOURCE
-:ljwrappers:classes UP-TO-DATE
-:copyWrappers UP-TO-DATE
-:rungenerated/Main.java:18: error: cannot find symbol
-        foo(x);
-        ^
-  symbol:   method foo(A)
-  location: class Main
-generated/Main.java:19: error: cannot find symbol
-        foo(y);
-        ^
-  symbol:   method foo(B)
-  location: class Main
-2 errors
-
-
-BUILD SUCCESSFUL
-
