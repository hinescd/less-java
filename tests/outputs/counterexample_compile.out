:generateGrammarSource UP-TO-DATE
:compileJava UP-TO-DATE
:processResources NO-SOURCE
:classes UP-TO-DATE
:ljwrappers:compileJava UP-TO-DATE
:ljwrappers:processResources NO-SOURCE
:ljwrappers:classes UP-TO-DATE
:copyWrappers UP-TO-DATE
:rungenerated/Main.java:18: error: cannot find symbol
        foo(x);
        ^
  symbol:   method foo(A)
  location: class Main
generated/Main.java:19: error: cannot find symbol
        foo(y);
        ^
  symbol:   method foo(B)
  location: class Main
2 errors


BUILD SUCCESSFUL

